docs/rules/no-deprecated-classnames.md

```diff
@@ -27,7 +27,7 @@ Examples of **correct** code for this rule:
 
 This rule accepts an object option, where the keys are the deprecated class names and the values are messages suggesting the updated class names to use instead. This allows teams to customize the rule based on their specific design system and TailwindCSS configuration.
 
-## Example
+## Example Configuration
```

Comment:
Aligning rule headings for "Example Configuration"

![Screenshot 2024-04-15 at 3 34 11 PM](https://github.com/MetaMask/eslint-plugin-design-tokens/assets/8112138/2a16c5f4-fa40-46a4-8782-cfe9524d85df)